### ISSUE TITLE
PTC Account

### ISSUE THREAD

**Issue by PabliniBis** [jabbink/PokemonGoBot#1258]
bot is not working. it tries to login, but it sends error

it was working ok with my google account, until that got banned.
first i used the same configuration file, but now im using the same as in template, just changing credentials and localization

<script src="//pastebin.com/embed_js/KnnpqSuY"></script>

im using the latest version from develop. running on win10 with java 1.8

I have read all the issues referring to login issues. none of them served me for good.
I can login well in app. and as i said, google account was working well

Leave this sentence in your issue as proof that you have read and used this template.

**Comment by EpicAwesomeWolf** [comment]
Can you give us an error message? This is wayyy too generic.

**Comment by EpicAwesomeWolf** [comment]
Oh wait, the Pastebin had it. Are you using the JSON template? If so, you made need to set it to PTC instead of Google.
`Caused by: java.lang.RuntimeException: java.util.concurrent.ExecutionException: com.google.protobuf.InvalidProtocolBufferException: Contents of buffer are null`

**Comment by PabliniBis** [comment]
im trying to use pastebin, but don't know how tu use.
and im using the config template, not JSON

**Comment by EpicAwesomeWolf** [comment]
I don't know if PTC is supported by the config template. Pinging @Sieberkev and @FrancYescO for clarification.

**Comment by PabliniBis** [comment]
now i tried with the JSON and get same error

**Comment by EpicAwesomeWolf** [comment]
@PabliniBis can you put a !!censored!! version of your JSON here?

**Comment by FrancYescO** [comment]
when this happen to me at login time 90% of time a restart will get it okay..

**Comment by EpicAwesomeWolf** [comment]
Yeah, it could have just gotten hung somewhere.

**Comment by Sieberkev** [comment]
> concurrent.ExecutionException: com.google.protobuf.InvalidProtocolBufferException: Contents of buffer are null

The underlaying API sends too much requests at once at the PoGo server. Restarting can be a solution, but the final solution will arrive when the new API from jabbink gets published for everyone.

**Comment by PabliniBis** [comment]
{
  "latitude" : 19.4392709,
  "longitude" : -99.2020797,
  "saveLocationOnShutdown" : true,
  "credentials" : {
    "type" : "ptc",
    "username" : "myptcname",
    "password" : "mypass"
  },
  "proxyServer" : "",
  "proxyPort" : "-1",
  "proxyType" : "SOCKS",
  "proxyUsername" : "",
  "proxyPassword" : "",
  "restApiPassword" : "",
  "speed" : 2.8,
  "randomSpeedRange" : 0.0,
  "followStreets" : [],
  "mapzenApiKey" : "",
  "googleApiKey" : "",
  "groupItemsByType" : false,
  "dropItems" : true,
  "uselessItems" : {
    "ITEM_REVIVE" : 20,
    "ITEM_MAX_REVIVE" : 10,
    "ITEM_POTION" : 0,
    "ITEM_SUPER_POTION" : 30,
    "ITEM_HYPER_POTION" : 50,
    "ITEM_MAX_POTION" : 50,
    "ITEM_POKE_BALL" : 40,
    "ITEM_GREAT_BALL" : 50,
    "ITEM_ULTRA_BALL" : 50,
    "ITEM_MASTER_BALL" : 10,
    "ITEM_RAZZ_BERRY" : 40,
    "ITEM_LUCKY_EGG" : -1,
    "ITEM_INCENSE_ORDINARY" : -1,
    "ITEM_TROY_DISK" : -1
  },
  "profileUpdateTimer" : 60,
  "timerWalkToStartPokestop" : -1,
  "randomNextPokestopSelection" : 5,
  "campLurePokestop" : -1,
  "desiredCatchProbability" : 0.4,
  "desiredCatchProbabilityUnwanted" : 0.0,
  "autotransfer" : true,
  "randomBallThrows" : false,
  "waitBetweenThrows" : false,
  "keepPokemonAmount" : 1,
  "maxPokemonAmount" : -1,
  "displayKeepalive" : true,
  "displayPokestopName" : false,
  "displayPokestopRewards" : true,
  "displayPokemonCatchRewards" : true,
  "displayIfPokemonFromLure" : true,
  "lootPokestop" : true,
  "catchPokemon" : true,
  "autoFillIncubator" : true,
  "sortByIv" : false,
  "desiredCurveRate" : 0.0,
  "neverUseBerries" : true,
  "allowLeaveStartArea" : false,
  "spawnRadius" : -1,
  "banSpinCount" : 0,
  "transferCpThreshold" : 400,
  "transferCpMinThreshold" : -1,
  "transferIvThreshold" : 80,
  "ignoredPokemon" : [ "EEVEE", "MEWTWO", "CHARMANDER" ],
  "obligatoryTransfer" : [ "DODUO", "RATTATA", "CATERPIE", "PIDGEY" ],
  "evolveBeforeTransfer" : [ "CATERPIE", "RATTATA", "WEEDLE", "PIDGEY" ],
  "evolveStackLimit" : 100,
  "useLuckyEgg" : 1,
  "export" : "",
  "guiPortSocket" : 8001,
  "initialMapSize" : 9,
  "waitChance" : 0.0,
  "waitTimeMin" : 0,
  "waitTimeMax" : 0
}

**Comment by PabliniBis** [comment]
I have restarted like  million times, it always gives same error. 
with google acc was working perfectly

**Comment by FrancYescO** [comment]
have you tried to reconfigure from a blank config file?

**Comment by PabliniBis** [comment]
yes. i have used it with all default, just my credentials and location

**Comment by MrBull** [comment]
You can try the following. Delete log, build, and bot-settings folders, then rebuild and rerun. When you get struck again, delete just only the bot-settings folder and rerun it again. This helps me all the time so far. Let me know what you get !

**Comment by staurakis** [comment]
Have you 'activated' your PTC account by clicking the related email PTC sent you?
I had the exact same issue. It let me play for some hours and then i received the above error. I think it's only an issue of PTC account activation from the email.

**Comment by PabliniBis** [comment]
i did as you said.
at first, same error but in the fifth attempt, it could login.
but now it says "got 0 pokestops"
then it decreases map size until it goes to 1
left it like that like 30 min
kept saying 0 pokestops

**Comment by FrancYescO** [comment]
Ip ban or too much Pokémon/pokestop softban
i did as you said.
at first, same error but in the fifth attempt, it could login.
but now it says "got 0 pokestops"
then it decreases map size until it goes to 1
left it like that like 30 min
kept saying 0 pokestops

**Comment by PabliniBis** [comment]
how could that be?
the account is relative new

**Comment by staurakis** [comment]
Have you checked how many pokestops have you harvested and how many pokemons have you catch in 24 hours period?
Have you changed ip to check?

**Comment by PabliniBis** [comment]
in the last 48 hrs i havent harvested pokestops
and i have tried connectin my laptop to mobile network instead of wifi
all same result

**Comment by staurakis** [comment]
Ok i understand. What is the current behavior of the official app? Cause i haven't seen any similar type of ban up to now....

**Comment by PabliniBis** [comment]
official app works perfectly
